systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/controllers/ConfigMapUtils.java

```diff
@@ -48,4 +50,25 @@ public static void waitForConfigMapLabelsChange(String namespaceName, String con
         }
     }
 
+    /**
+     * Creates a ConfigMap in a specified Kubernetes namespace.
+     *
+     * @param namespaceName     the name of the Kubernetes namespace where the ConfigMap will be created
+     * @param configMapName     the name of the ConfigMap to be created
+     * @param dataKey           the key for the data to be stored in the ConfigMap
+     * @param dataValue         the value associated with the key in the ConfigMap
+     */
+    public static void createConfigMap(String namespaceName, String configMapName, String dataKey, String dataValue) {
```

Comment:
I think that you can move this to some `ConfigMapTemplates` class?
Also in order to properly delete the resource, I think that you want to do it via ResourceManager.

Reply:
Sure, good point. Thanks.